workflow_scripts/markdownlint-check.sh

```
@@ -0,0 +1,22 @@
+
+if files=$(git diff --name-only --diff-filter=ACMRT "${MARKDOWNLINT_BASE}...HEAD" -- '*.md' 2>/dev/null); then
+    if [ -n "${files}" ]; then
+        status=0
+        while IFS= read -r file; do
+            ${MDLINT} "${file}" || status=1
+        done <<EOF
+${files}
+EOF
+        exit "${status}"
+    else
+        find . -type f -name '*.md' -not -path './target/*' -exec ${MDLINT} {} +
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)